modelexpress_client/python/modelexpress/vllm_loader.py

```
@@ -0,0 +1,1040 @@
+        _log(f"[Worker {device_id}] [TIMING] Registering target tensors with NIXL...", "DEBUG")
+        self._nixl_manager.register_tensors(target_tensors)
+        reg_time = _time.perf_counter() - t0
+        _log(f"[Worker {device_id}] [TIMING] Target tensors registered in {reg_time:.3f}s", "INFO")
+
+        # COORDINATION: Wait for NIXL ready flag before initiating transfer
+        # The nixl_ready flag is only set AFTER:
+        # 1. vLLM health endpoint returns 200
+        # 2. 30s grace period for system stabilization
+        # 3. Successful test inference proves stability
+        _log(f"[Worker {device_id}] Waiting for source NIXL ready (includes stability verification)...", "INFO")
+        source_ready, cached_session_id, cached_metadata_hash = SourceReadyCoordinator.wait_for_source_ready(
```

Ruff (0.14.14)

[warning] Unpacked variable `cached_metadata_hash` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)